packages/react-native/src/NotifeeApiModule.ts

```diff
@@ -350,6 +350,20 @@ export default class NotifeeApiModule extends NotifeeNativeModule implements Mod
     });
   };
 
+  public canCreateTimeStampTrigger = (): Promise<boolean> => {
+    if (isAndroid) {
+      return this.native.canCreateTimeStampTrigger();
```

Comment:
@vincent-paing thank you so much for this contribution. I will be looking at this p/r closer over the next few days. But, one small thing can we make `timestamp` all lowercase... e.g. `canCreateTimestampTrigger`